src/main/java/kr/allcll/backend/domain/seat/GeneralSeatSender.java

```diff
@@ -0,0 +1,52 @@
+package kr.allcll.backend.domain.seat;
+
+import java.time.Duration;
+import java.util.List;
+import kr.allcll.backend.domain.seat.dto.SeatDto;
+import kr.allcll.backend.domain.seat.dto.SeatsResponse;
+import kr.allcll.backend.support.schedule.ScheduledTaskHandler;
+import kr.allcll.backend.support.sse.SseService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class GeneralSeatSender {
+
+    private static final String EVENT_NAME = "nonMajorSeats";
```

Comment:
이 이벤트 네임도 수정해야하지 않을까요?!
```suggestion
    private static final String EVENT_NAME = "generalSeats";
```

Reply:
이거는 클라이언트한테 직접 전달되는 이벤트명이에요! 그래서 변경하게 되면 클라이언트와 맞춰야 해서 우선은 납뒀어요.